bundles/binding/org.openhab.binding.octoller/src/main/java/org/octoller/devicecom/Connection.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+/**
+ ********************************************************************************************************
+ * openHAB binding for octoller (www.octoller.com)
+ * For use with octoller-Gateway
+ ********************************************************************************************************
+ */
+
+package org.octoller.devicecom;
+
+import java.io.*;
+import java.net.*;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+import org.openhab.binding.octoller.internal.octollerBindingConfig;
+import org.openhab.core.events.EventPublisher;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.library.types.PercentType;
+import org.openhab.core.types.State;
+
+/**
+ * @author JPlenert
+ * @since 1.5.1
+ * openHAB binding for octoller (www.octoller.com)
+ * For use with octoller-Gateway
```

Comment:
move comment above @author/@version tag according to the java coding conventions (see https://github.com/openhab/openhab/wiki/How-To-Contribute)